src/main.zig

```diff
@@ -55,6 +55,8 @@ pub const max_src_size = std.math.maxInt(u32);
 
 pub const debug_extensions_enabled = builtin.mode == .Debug;
 
+pub const side_channels_mitigations = .none;
```

Comment:
What is this doing here? I don't see that this is read anywhere.

Reply:
This is to play well with https://github.com/ziglang/zig/pull/13739 - For the compiler itself, we can prioritize performance over side channels mitigations.

Reply:
The global has been removed.